Reverse sense of difference

rscottbailey opened the pull request:
Fixes #32.

All of the other difference producers used the correct ordering.

rscottbailey committed:
Reverse sense of difference

Fixes #32.

tarkatronic commented:
@rscottbailey Thanks for the patch! Looks like the tests need to be updated to match this. If you can take care of that, we can get this merged in. :-)

rscottbailey committed:
Update unit tests

Update expected call trace to track exchange of commit roles when generating
differences.

rscottbailey commented:
I added a second commit that makes the unit tests work again.
